### DISCUSSION TITLE
[Question]: how to define a url prefix past?

### DISCUSSION THREAD

**Discussion by dportabella** [danny-avila/LibreChat#4595] [Q&A]
### What is your question?

I'd like to install the chatbot at something like https://example.com/version1/chatbot
And what is the difference between DOMAIN_CLIENT and DOMAIN_SERVER?
```
DOMAIN_CLIENT=http://localhost:3080
DOMAIN_SERVER=http://localhost:3080

```
Is it possible to specify the url path prefix here? (I tried, but it is ignore):
```
DOMAIN_CLIENT=https://example.com:80/version1/chatbot
DOMAIN_SERVER=https://example.com:80/version1/chatbot

```

### More Details

-

### What is the main subject of your question?

Installation

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by dportabella** [comment]
I no longer need the url path prefix, i will use https://chatbot.versions1.example.com:80/
however, can you explain what are the parameters `DOMAIN_CLIENT`, `DOMAIN_SERVER`, and in which context they are used?

**Reply by danny-avila** [reply]
They are simply used to construct valid links for OAuth2 purposes and Emailing services (account signup, password reset, etc.).

**Reply by fbrcosta** [reply]
Hello @danny-avila, i came across the same requirement. I made an analysis on the code and i didn't find any parameter to set this. But i wanted to make sure i havent missed anything. 
If this feature is not supported, i would be happy to contribute.
